frontend/src/metabase/styled-components/theme/css-variables.ts

```diff
@@ -41,9 +41,10 @@ export function getMetabaseSdkCssVariables(theme: MantineTheme, font: string) {
   return css`
     :root {
       --mb-default-font-family: ${font};
+      ${createColorVars("light")}
```

Comment:
we're missing this. Previously, it was included via `colors.module.css` but it seems we've moved those to `frontend/src/metabase/lib/colors/colors.ts` which is used in this `createColorVars` function.

Reply:
The difference was that `colors.module.css` was included via PostCSS plugin, so it wasn't that obvious bofer, but it is very clear here.